pkg/awsManager/vpcVpnManager.go

```diff
@@ -317,29 +305,24 @@ func DeleteGateway(client clientpkg.Client, vpcID *string) error {
 			break
 		}
 	}
-
 	return nil
 }
 
-func DeleteSubnetsForVPC(client clientpkg.Client, vpcId *string) error {
-
+func deleteSubnetsForVPC(client clientpkg.Client, vpcId *string, logger logr.Logger) error {
```

Comment:
why was this unexported?

Reply:
because it is a helper method that isn't and shouldn't be used outside this file